tests/cover/test_timeout.py

```diff
@@ -92,7 +92,7 @@ def test_slow_failing_test_4(x):
 strict_timeout_settings = default_timeout_settings
 
 
-@fails_with(DeprecationWarning)
+@fails_with(HypothesisDeprecationWarning)
```

Comment:
It would probably make sense to replace this with a `@checks_deprecated_behaviour` instead.